### PR TITLE
[MAINTENANCE] Update great_expectations_contrib CLI tool to use existing diagnostic classes

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Use existing ExpectationDiagnostics class in manifest
- Update build script to work with existing contrib packages

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: assets/scripts/build_package_gallery.py
Comment (kenwade4) [inline assets/scripts/build_package_gallery.py]: I'm not sure how the pip package is meant to be used in code, but I got an error `no such option: -r` when running build_package_gallery.py

```
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.

Usage:
  build_package_gallery.py <command> [options]

no such option: -r
```

File: package_manifests.json
Comment (cdkini) [inline package_manifests.json]: @kenwade4 I'm only include this here to show you what the output of this script is (as long as Expectations are exported in the __init__.py)

File: contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/expectations/__init__.py
Comment (cdkini) [inline contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/expectations/__init__.py:3]: @kenwade4 I modified this just to get the output noted in the `package_manifest.json` file.

I'm happy to keep this here but let me know if we should remove and defer to the Capital One team.

Comment (kenwade4) [reply contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/expectations/__init__.py:3]: This change should be kept here for sure

File: package_manifests.json
Comment (kenwade4) [reply package_manifests.json]: Just remember to delete this before merging

**Review by kenwade4 — APPROVED** [review]

File: package_manifests.json
Comment (cdkini) [reply package_manifests.json]: done!

**Pull request merged**
